frontend_tests/node_tests/search_suggestion.js

```diff
@@ -459,6 +462,39 @@ run_test('empty_query_suggestions', () => {
     assert.equal(describe('is:alerted'), 'Alerted messages');
     assert.equal(describe('is:unread'), 'Unread messages');
     assert.equal(describe('sender:bob@zulip.com'), 'Sent by me');
+    assert.equal(describe('has:link'), 'Messages with one or more link');
+    assert.equal(describe('has:image'), 'Messages with one or more image');
+    assert.equal(describe('has:attachment'), 'Messages with one or more attachment');
+});
+
+run_test('has_operator_suggestions', () => {
+    // Checks that category wise suggestions are displayed instead of a single
```

Comment:
Although it is checked in the test before this i.e `empty_query_suggestions` that category wise has operators are shown, changes to that test may push the has suggestions down as they have a low priority. That is why a second test was added.